apps/gql/src/schema.ts

```diff
@@ -0,0 +1,39 @@
+import { IResolvers } from '@graphql-tools/utils'
+import { getForecasts, getLocations } from './forecast';
+
+export const typeDefs = /* GraphQL */ `
+  type Query {
+    hello: String
+    forecasts(latitude: Float!, longitude: Float!, timezone: String): [WeatherForecast]
+    locations(name: String!): [Location!]
+  }
+
+  type WeatherForecast {
+    date: String!
+    minimum: Float!
+    maximum: Float!
+    weatherCode: Int!
```

Comment:
Prop will rework to be an enum with values like sunny, mostly sunny, partly cloudy, rainy etc